.github/workflows/inference_cache_stable_diffusion.yml

```diff
@@ -0,0 +1,52 @@
+name: Optimum neuron SD inference cache builder
+
+on:
+  workflow_dispatch:
+  schedule:
+    # Schedule the workflow to run every Saturday at midnight UTC
+    - cron: '0 0 * * 6'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+
+jobs:
+  cache:
+    name: Create optimum-neuron inference cache
+    runs-on:
+      group: aws-inf2-8xlarge
+    env:
+      AWS_REGION: us-east-1
+    strategy:
+      fail-fast: false
+      matrix:
+        config: [stable-diffusion]
+    steps:
+      - name: Install Neuron runtime
+        run: |
+          . /etc/os-release
+          sudo tee /etc/apt/sources.list.d/neuron.list > /dev/null <<EOF
+          deb https://apt.repos.neuron.amazonaws.com ${VERSION_CODENAME} main
+          EOF
+          wget -qO - https://apt.repos.neuron.amazonaws.com/GPG-PUB-KEY-AMAZON-AWS-NEURON.PUB | sudo apt-key add -
+          sudo apt-get update -y
+          sudo apt-get install aws-neuronx-tools=2.18.3.0 aws-neuronx-runtime-lib=2.21.41.0-fb1705f5f aws-neuronx-collectives=2.21.46.0-69b77134b  -y
```

Comment:
These packages are obsolete: you should use the ones from 2.19.1 (this is true also for the llm cache workflow)

Reply:
How are these obsolete? The versions in `inference_cache_stable_diffusion.yml` are version numbers I copied from my local 2.19.1 env, do you mean that I shall use the[ ones in AWS Neuron doc](https://awsdocs-neuron.readthedocs-hosted.com/en/latest/release-notes/index.html#trn1-packages), those without the suffix?

Reply:
The versions in the LLM workflow files seem to come from [Neuron 2.18.2](https://awsdocs-neuron.readthedocs-hosted.com/en/latest/release-notes/prev/content.html#neuron-2-18-2-04-25-2024).

Reply:
You're right: the versions you used are correct. It is only those of the LLM workflow that have not been updated.